packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableSelector.ts

```diff
@@ -84,3 +86,18 @@ function setSelectorDivPosition(context: DragAndDropContext, trigger: HTMLElemen
         trigger.style.left = `${rect.left - TABLE_SELECTOR_LENGTH - 2}px`;
     }
 }
+
+function isSelectorInsideEditor(
+    editor: IEditor,
+    rect: Rect | null,
+    contentDiv?: EventTarget
+): boolean {
+    const visibleViewport = editor.getVisibleViewport();
+    if (contentDiv && safeInstanceOf(contentDiv, 'HTMLElement') && visibleViewport && rect) {
+        const containerRect = normalizeRect(contentDiv.getBoundingClientRect());
+
+        return containerRect.top <= rect.top && visibleViewport.top <= rect.top;
```

Comment:
Do we need to check bottom?

Reply:
I think for Table Selector we only need to check if the top of the table is visible, renamed the function name from `isSelectorInsideEditor `to `isTableTopVisible ` since the name was confusing.